client/rest-high-level/src/test/java/org/elasticsearch/client/ml/dataframe/ClassificationTests.java

```diff
@@ -42,7 +42,7 @@ public static Classification randomClassification() {
             .setEta(randomBoolean() ? null : randomDoubleBetween(0.001, 1.0, true))
             .setMaxTrees(randomBoolean() ? null : randomIntBetween(1, 2000))
             .setFeatureBagFraction(randomBoolean() ? null : randomDoubleBetween(0.0, 1.0, false))
-            .setNumTopFeatureImportanceValues(randomBoolean() ? null : randomIntBetween(0, Integer.MAX_VALUE))
+            .setNumTopFeatureImportanceValues(randomBoolean() ? null : randomInt())
```

Comment:
Why was this changed?

Reply:
I wanted the client code to be able to pass `-1` value through.

Reply:
But this is for `numTopFeatureImportanceValues`.

Reply:
Ooops, shame on me :(
Fixed.